scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#to be executed from the root project directory
+git rev-parse --short HEAD > version.txt
```

Comment:
The version that lands in the `version.txt` file is connected more with the version of the UI and only the UI uses it so maybe it should land in the `shiny` directory?